tests/unit/via/views/test_route_by_content.py

```diff
@@ -17,52 +17,23 @@
 
 
 class TestRouteByContent:
-    @pytest.mark.parametrize(
-        "requested_path,expected_location,content_type",
-        [
-            # If the requested pdf URL has no query string then it should just
-            # redirect to the requested URL, with no query string (but with the
-            (
-                "/https://example.com/foo",
-                "http://localhost/pdf/https://example.com/foo",
-                "application/pdf",
-            ),
-            # If the requested pdf URL has a query string then the query string
-            # should be preserved in the URL that it redirects to.
-            (
-                "/https://example.com/foo?bar=baz",
-                "http://localhost/pdf/https://example.com/foo?bar=baz",
-                "application/pdf",
-            ),
-            # If the requested html URL has a query string then the query string
-            # should be preserved in the URL that it redirects to.
-            (
-                "/https://example.com/foo?bar=baz",
-                "http://via.hypothes.is/https://example.com/foo?bar=baz",
-                "text/html",
-            ),
-            # If the requested html URL has a client query params then the query params
-            # should be preserved in the URL that it redirects to.
-            (
-                "/https://example.com/foo?via.open_sidebar=1",
-                "http://via.hypothes.is/https://example.com/foo?via.open_sidebar=1",
-                "text/html",
-            ),
-        ],
-    )
-    def test_redirect_location(
-        self, make_pyramid_request, requested_path, expected_location, content_type
-    ):
-
+    def test_we_do_not_lookup_via_parameters(self, make_pyramid_request, requests):
```

Comment:
The test name confused me. This is testing that we don't pass the `via.*` params to the third-party server, right? Maybe `test_we_dont_send_via_params_to_third_parties`?

Reply:
Oh god. I've just spotted this whole test is replicated again lower down. So the first test was already duplicating things in here twice. I've just removed mine entirely and left the other there.